scripts/hauler_manifest.yaml

```diff
@@ -0,0 +1,70 @@
+# This is a base file to be used to import all the required resources to run
```

Comment:
Quibble: does it make sense to drop the `hauler_manifest.yaml` in scripts? Maybe it's better suited under `charts/` (besides the README.md), or in a new folder that signifies that it is a deliverable.

Reply:
> Quibble: does it make sense to put the `hauler_manifest.yaml` in `scripts/`? Maybe it's better suited under `charts/` (besides the README.md), or in a new folder that signifies that it is a deliverable.

I agree on this one, it's not really a script.